src/main/webapp/edit-cache/edit-cache.controller.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+angular.module('managementConsole')
+  .controller('editCacheCtrl', [
+    '$scope',
+    '$state',
+    '$stateParams',
+    'utils',
+    'modelController',
+    function ($scope, $state, $stateParams, utils, modelController) {
+      //-- Variables --//
+
+      if (!modelController.isAuthenticated()) {
```

Comment:
While this also works, it would be better to check for authentication before entering controller, ideally on route change start! I think this should not be solved in this PR, but I suggest creating an issue for this, so it can be done later.